packages/in_app_purchase/example/pubspec.yaml

```diff
@@ -14,6 +14,7 @@ dev_dependencies:
     sdk: flutter
   in_app_purchase:
     path: ../
+  e2e: ^0.2.0
```

Comment:
Does this also need a min Flutter SDK version to build the example app wtih? For the other plugins I've migrated I generally needed to up this to the latest stable, even though the plugin itself just needed 1.6.7.